ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp

```diff
@@ -0,0 +1,116 @@
+// [WriteFile Name=ExploreScenesInFlyoverAR, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "ExploreScenesInFlyoverAR.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "IntegratedMeshLayer.h"
+
+using namespace Esri::ArcGISRuntime;
+using namespace Esri::ArcGISRuntime::Toolkit;
```

Comment:
totally optional, but one thing to consider could be using a namespace alias to make it very clear that some of these objects come from the toolkit. `namespace toolkit = Esri::ArcGISRuntime::Toolkit;`

Reply:
I like this idea. Helps to highlight the objects coming from the toolkit.